code/src/Components/Button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+//props title className and function???
+const Button = (props) => {
+    return (
+        <button
+            className="button"
```

Comment:
I think choosing on of "" or '' will make your code even cleaner in the future (mine as well :D) or have a clear structure when using "" and ''.

Reply:
Hahah yees, I will work on that 😄